store/hbase/snapshot.go

```diff
@@ -72,6 +72,50 @@ func (s *hbaseSnapshot) Get(k kv.Key) ([]byte, error) {
 	return nil, kv.ErrNotExist
 }
 
+// BatchGet implements kv.Snapshot.BatchGet().
+func (s *hbaseSnapshot) BatchGet(keys []kv.Key) (map[string][]byte, error) {
+	gets := make([]*hbase.Get, len(keys))
+	for i, key := range keys {
+		g := hbase.NewGet(key)
+		g.AddColumn([]byte(ColFamily), []byte(Qualifier))
```

Comment:
you should define a slice variable, no need to convert the const string every time.